scripts/migrate-advanced.js

```
@@ -0,0 +1,106 @@
+      // 保守策略：设置为过期+几天前，让真正无用的key在下次扫描时被检测出来
+      const conservativeTime = now - (expirationDays + 3) * 24 * 60 * 60 * 1000;
+      return conservativeTime;
+      
+    case 'staggered':
+      // 分散策略：根据key的哈希值分散时间，避免所有key同时过期
+      const hash = simpleHash(key + modulePath);
+      const randomDays = hash % expirationDays; // 0 到 expirationDays-1 天前
+      const staggeredTime = now - randomDays * 24 * 60 * 60 * 1000;
+      return staggeredTime;
+      
+    default:
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)